docs/portal-builder/maps/discarded.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Discarded
+---
+
+{{< toc >}}
```

Comment:
should we tell upfront which layouts are not available? as currently one needs to scroll down to find if a layout is available or not

Reply:
yes, definitely, but I would keep it as an enhancement. I wanted to do this PR to close this topic for now on my end :D